src/js/modules/infragistics.ui.editors.js

```diff
@@ -12059,5 +12062,682 @@
+				this.options.dateInputFormat = this.options.timeInputFormat;
+			} else {
+				this.options.dateInputFormat = "time";
+			}
+
+			this._super();
+
+			if (this.options.minTimeValue) {
+				this.options.minValue = this._parseDateFromMaskedValue(this.options.minTimeValue);
+			}
+
+			if (this.options.maxTimeValue) {
+				this.options.maxValue = this._parseDateFromMaskedValue(this.options.maxTimeValue);
+			}
+
+			var buttons = this.options.buttonType.toString().split(/[\s,]+/);
+
+			if ($.inArray("dropdown", buttons) === 0) {
+				this._populateDropDown(this.options.minTimeValue, this.options.maxTimeValue);
+			}
+		},
+		getSelectedListItem: function () {
+			/* Gets the selected list item.
+			```
+			var selectedItem =  $(".selector").%%WidgetName%%("getSelectedListItem");
+			```
+				returnType="$" Selected list item.*/
+			return $.ui.igTextEditor.prototype.getSelectedListItem.call(this);
+		},
+		dropDownVisible: function () {
+			/* Returns the visibility state of the calendar.
+			```
+			$(".selector").igTimePicker("dropDownVisible");
+			```
+				returnType="bool" The visibility state of the calendar. */
+			return $.ui.igTextEditor.prototype.dropDownVisible.call(this);
+		},
+		dropDownButton: function () {
+			/* Returns a reference to the drop-down button UI element of the editor.
+			```
+			var button = $(".selector").%%WidgetName%%("dropDownButton");
+			```
+				returnType="$" Returns reference to jquery object. */
+			return $.ui.igTextEditor.prototype.dropDownButton.call(this);
+		},
+		dropDownContainer: function () {
+			/* Gets reference to jquery object which is used as container of drop-down list.
+			```
+				 $(".selector").%%WidgetName%%("dropDownContainer");
+			```
+				returnType="$" Returns reference to jquery object. */
+			return $.ui.igTextEditor.prototype.dropDownContainer.call(this);
+		},
+		findListItemIndex: function (text, matchType) {
+			/* Finds index of list item by text that matches with the search parameters.
+			```
+				var item =  $(".selector").igTimePicker("findListItemIndex");
+
+			```
+				paramType="string" optional="false" The text to search for in the drop down list.
+				paramType="startsWith|endsWith|contains|exact" optional="true" The rule that is applied for searching the text.
+				returnType="number" Returns index of the found item. */
+			return $.ui.igTextEditor.prototype.findListItemIndex.call(this, text, matchType);
+		},
+		selectedListIndex: function (index) {
+			/* Gets the index of the selected list item. Sets selected item by index.
+			```
+				$(".selector").%%WidgetName%%("selectedListIndex", 1);
+				var selectedIndex = $(".selector").%%WidgetName%%("selectedListIndex");
+			```
+				paramType="number" optional="true" The index of the item that needs to be selected.
+				returnType="number" Returns the selected index. */
+			return $.ui.igTextEditor.prototype.selectedListIndex.call(this, index);

```

Comment:
Use `_super` instead. Btw, do you really need the overrides, since they do nothing different?